Formula/woof.rb

```diff
@@ -1,16 +1,20 @@
 class Woof < Formula
   desc "Ad-hoc single-file webserver"
   homepage "http://www.home.unix-ag.org/simon/woof.html"
-  url "http://www.home.unix-ag.org/simon/woof-2012-05-31.py"
-  version "20120531"
-  sha256 "d84353d07f768321a1921a67193510bf292cf0213295e8c7689176f32e945572"
+  url "https://github.com/simon-budig/woof/archive/woof-20220202.tar.gz"
+  sha256 "cf29214aca196a1778e2f5df1f5cc653da9bee8fc2b19f01439c750c41ae83c1"
+  license "GPL-2.0-or-later"
 
   bottle do
     sha256 cellar: :any_skip_relocation, all: "854310e14cd958d44e42c8b6dd5aac3ca360ae2154763a0967cfbe4dee76a512"
   end
 
+  depends_on "python@3.10"
+
   def install
-    bin.install "woof-2012-05-31.py" => "woof"
+    inreplace "woof", "/usr/bin/env python3", Formula["python@3.10"].opt_bin/"python3"
```

Comment:
We actually have a helper for this:
```suggestion
    rewrite_shebang detected_python_shebang, "woof"
```

You'll also need to `include Language::Python::Shebang`

Reply:
I had done some cursory `grep`ing to try to find relevant patterns but, unfortunately, didn't encounter one using this helper. Thanks for the heads up!